packages/actor-init-query/lib/HttpServiceSparqlEndpoint.ts

```diff
@@ -13,20 +11,20 @@ import { ActionContext } from '@comunica/core';
 import type { ICliArgsHandler, QueryQuads, QueryType } from '@comunica/types';
 import type * as RDF from '@rdfjs/types';
 import { ArrayIterator } from 'asynciterator';
-import yargs from 'yargs';
 // eslint-disable-next-line import/no-useless-path-segments
 import { QueryEngineBase, QueryEngineFactoryBase } from '..';
 // eslint-disable-next-line import/no-useless-path-segments
 import type { IDynamicQueryEngineOptions } from '..';
 import { CliArgsHandlerBase } from './cli/CliArgsHandlerBase';
 import { CliArgsHandlerHttp } from './cli/CliArgsHandlerHttp';
 
+const clusterUntyped = require('cluster');
 const process: NodeJS.Process = require('process/');
-
 const quad = require('rdf-quad');
+const yargs = require('yargs').default;
```

Comment:
This also causes yargs to lose its types, and can be fixed by adjusting the constructor arguments based on my testing (same as the comment above). Other changes are fine I think.